tsconfig.build.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "sourceMap": false
```

Comment:
Any particular reason for disabling source maps? It makes debugging more painful than necessary, and there's not really anything I can come up with that would benefit when disabling them.

Reply:
agree